app/helpers/interest_icons_helper.rb

```diff
@@ -0,0 +1,85 @@
+  # Draw the cutesy eye icons in the upper right side of screen.  It does it
```

Comment:
New file has 8 rubocop "offenses".  I recognize that you're just moving code, so again not a blocker.  Personally I try to clean things up when I make moves like these.